PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/AzureActionsComponent.java

```diff
@@ -78,12 +82,16 @@
     public AzureActionsComponent() {
         DefaultLoader.setPluginComponent(this);
         DefaultLoader.setUiHelper(new UIHelperImpl());
-        DefaultLoader.setIdeHelper(new IDEHelperImpl());
-        Node.setNode2Actions(NodeActionsMap.node2Actions);
+        DefaultLoader.setIdeHelper(createIDEHelper());
         SchedulerProviderFactory.getInstance().init(new AppSchedulerProvider());
         MvpUIHelperFactory.getInstance().init(new MvpUIHelperImpl());
 
-        HDInsightLoader.setHHDInsightHelper(new HDInsightHelperImpl());
```

Comment:
There're lots of Java dependencies in HDInsight module, but it should also be enabled for all IDEs.

Reply:
To be honest I'd keep it disabled for this specific PR.

HDInsight code relies heavily on IntelliJ / Java features that are not available in Rider (or other IDE's for that mater). One example would be the HDInsights actions expect a Java project model, the ability to create a `.war` file, ...

Perhaps, when/if this PR gets in, good to log an issue on the backlog that would describe desired HDInsight functionality for e.g. Rider?